src/main/java/de/adesso/kicker/user/User.java

```diff
@@ -100,4 +101,20 @@ public String toString() {
         return "User{" + "userId='" + userId + '\'' + ", firstName='" + firstName + '\'' + ", lastName='" + lastName
                 + '\'' + ", email='" + email + '\'' + ", wins=" + wins + ", losses=" + losses + '}';
     }
+
+    @Override
+    public boolean equals(Object o) {
```

Comment:
Sollte hier nicht auch ein User Objekt übergeben werden?

Reply:
Die Methode equals wird nur überschrieben, d.h alles muss identisch sein außer der Funktionskörper da verwechselst du eventuell etwas mit überladen, zur Not siehe "07 - Collections und Generics" Folie 55. Da wird ebenfalls für die Klasse Kunde die equals Methode überschrieben, ebenfalls mit einem Object.